app/shared/inputs/TextField.js

```diff
@@ -30,17 +27,21 @@ export default function TextField({ endAdornments, ...restProps }) {
   return (
     <MuiTextField
       variant="outlined"
-      InputProps={{
-        endAdornment: endAdornments?.length > 0 && (
-          <InputAdornment position="end">
-            {endAdornments.map(
-              (adornment) => ADORNMENTS[adornment] ?? adornment,
-            )}
-          </InputAdornment>
-        ),
-        ...restProps?.InputProps, // ensure adornment shorthand doesn't override any other InputProps
-      }}
       {...restProps}
+      slotProps={{
+        ...restProps.slotProps,
+        input: {
+          sx: { fontFamily: 'var(--outfit-font)' },
+          endAdornment: endAdornments?.length > 0 && (
+            <InputAdornment position="end">
+              {endAdornments.map(
+                (adornment) => ADORNMENTS[adornment] ?? adornment,
+              )}
+            </InputAdornment>
+          ),
+          ...restProps.slotProps?.input,
+        },
+      }}
```

Comment:
Same behavior, just via `slotProps`. InputProps is actually deprecated (and didn't support overriding the style the way we needed to).